### PR TITLE
Geopy-1400

### PR DESCRIPTION
**Pull request by domfournier**

### CODE REVIEW

File: docs/source/edge_detection.rst
Comment (benk-mira) [inline docs/source/edge_detection.rst:16]: Link is broken

File: docs/source/edge_detection.rst
Comment (benk-mira) [reply docs/source/edge_detection.rst:43]: I find this confusing.  Took a stab at clearing it up, but will defer to you.

File: docs/source/edge_detection.rst
Comment (benk-mira) [inline docs/source/edge_detection.rst:88]: link didn't work

File: docs/source/edge_detection.rst
Comment (benk-mira) [inline docs/source/edge_detection.rst:171]:
```suggestion
Note that shorter lines are filtered when increasing the ``line length`` parameter.
```

File: docs/source/edge_detection.rst
Comment (benk-mira) [inline docs/source/edge_detection.rst:176]:
```suggestion
The following figure shows the effect of increasing the ``line gap`` parameter in the Hough Line Transform. For ``line gap=1`` and ``line gap=10`` we obtain:
```

File: docs/source/edge_detection.rst
Comment (benk-mira) [inline docs/source/edge_detection.rst:208]: Maybe put the parameters image above to give the side-by-side more space (hard to see).

File: docs/source/trend_lines.rst
Comment (benk-mira) [inline docs/source/trend_lines.rst:13]: Broken link

File: docs/source/trend_lines.rst
Comment (benk-mira) [inline docs/source/trend_lines.rst:38]: Weird indentation

File: docs/source/trend_lines.rst
Comment (benk-mira) [inline docs/source/trend_lines.rst:82]: Broken link

File: docs/source/trend_lines.rst
Comment (benk-mira) [inline docs/source/trend_lines.rst:56]: The images are a nice touch! what is the significance of the blue and red dashed lines?

File: docs/source/trend_lines.rst
Comment (benk-mira) [inline docs/source/trend_lines.rst:101]: I would give the images more space by organizing vertically

File: docs/source/trend_lines.rst
Comment (benk-mira) [inline docs/source/trend_lines.rst:90]: Point to release branch that will contain the asset described (empty on main until we merge)

File: docs/source/trend_lines.rst
Comment (benk-mira) [inline docs/source/trend_lines.rst:121]: I would align these vertically too, with last sentence in between the two images.

File: docs/source/getting_started.rst
Comment (benk-mira) [inline docs/source/getting_started.rst:38]:
```suggestion
    In any case, the installation of **curve-apps** forces the usage of the conda-forge repository,
```

File: docs/source/getting_started.rst
Comment (benk-mira) [inline docs/source/getting_started.rst:41]:
```suggestion
Download the latest curve-apps
```

File: docs/source/getting_started.rst
Comment (benk-mira) [inline docs/source/getting_started.rst:59]:
```suggestion
The same batch file can be used to install or update **curve-apps**.
```

**Review by benk-mira — CHANGES_REQUESTED** [review]
Just noticed these on my docs pr..

File: docs/source/edge_detection.rst
Comment (domfournier) [reply docs/source/edge_detection.rst:16]: These won't work until we are fully build

File: docs/source/edge_detection.rst
Comment (domfournier) [reply docs/source/edge_detection.rst:53]: Line gap is a detection parameter on pixels, this one merges the resulting vertices to reduce complexity. I re-worded a bit.

File: docs/source/edge_detection.rst
Comment (domfournier) [reply docs/source/edge_detection.rst:68]: I get your point. Curves is the final product, made up of many "segments".

File: docs/source/edge_detection.rst
Comment (domfournier) [reply docs/source/edge_detection.rst:88]: Fixed

File: docs/source/edge_detection.rst
Comment (domfournier) [reply docs/source/edge_detection.rst:208]: Done

File: docs/source/trend_lines.rst
Comment (domfournier) [reply docs/source/trend_lines.rst:38]: Shifted

File: docs/source/trend_lines.rst
Comment (domfournier) [reply docs/source/trend_lines.rst:56]: Re-edited for clarity

File: docs/source/trend_lines.rst
Comment (domfournier) [reply docs/source/trend_lines.rst:90]: I would rather point to the stable version. Ok if broken at first

File: docs/source/edge_detection.rst
Comment (domfournier) [reply docs/source/edge_detection.rst:94]: I would rather not to have to edit those paths again. We can have them broken at first

**Review by benk-mira — APPROVED** [review]
Better still!

**Pull request merged**
